site/docs/v1/tech/messengers/kafka-messenger.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: doc
+title: Kafka Messenger
+description: Adding and Configuring Messengers - Kafka
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
```

Comment:
Kafka wasn't premium in the past. Is it premium now? I don't think so https://github.com/FusionAuth/fusionauth-app/blob/master/src/main/java/io/fusionauth/app/action/api/MessengerAction.java 

In fact I don't think any of these are premium features. You can create messengers without a license, you just can't use them for MFA.